tests/Makefile.am

```diff
@@ -17,7 +17,7 @@ EXTRA_DIST = Makefile.am \
 	rrdcreate \
 	tune1-testa-mod1.dump tune1-testa-mod2.dump tune1-testorg.dump \
 	tune2-testa-mod1.dump tune2-testorg.dump \
-	valgrind-supressions dcounter1 dcounter1.output graph1.output vformatter1 rpn1.output rpn2.output \
+	valgrind-supressions dcounter1 dcounter1.output graph1.output group2.output vformatter1 rpn1.output rpn2.output \
```

Comment:
The typo is still not fixed (it is just a different typo since fc9027e):
`group2.output -> graph2.output`
The file is called **`graph2.output`**, otherwise `make dist` will fail.